newton/utils/render.py

```diff
@@ -481,7 +483,302 @@ class SimRendererUsd(CreateSimRenderer(renderer=UsdRenderer)):
             renderer.save()  # Save the USD file
     """
 
-    pass
+    def __init__(
+        self,
+        model: newton.Model,
+        stage,
+        source_stage=None,
```

Comment:
I think it is possible to set type hints for `Usd.Stage` here, like you did for some of the other methods?